fvm/src/machine/default.rs

```diff
@@ -92,6 +100,13 @@ where
             ));
         }
 
+        let empty_arr_cid = blockstore.put(Blake2b256, &EMPTY_ARRAY_BLOCK)?;
+
+        debug_assert!(
+            empty_arr_cid == *EMPTY_ARR_CID,
+            "empty CID sanity check failed",
+        );
```

Comment:
Helper "put empty blocks" function? With a comment explaining why we do this because this is really non-obvious?

(but, like, lowest priority ever)

Reply:
done